.github/workflows/test-package.yml

```
@@ -20,21 +20,20 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        sdk: [dev]
+        sdk: [2.12.0, dev]
     steps:
       - uses: actions/checkout@v2
-      - uses: dart-lang/setup-dart@v0.3
+      - uses: dart-lang/setup-dart@v1.0
         with:
           sdk: ${{ matrix.sdk }}
       - id: install
-        name: Install dependencies
         run: dart pub get
-      - name: Check formatting
-        run: dart format --output=none --set-exit-if-changed .
-        if: always() && steps.install.outcome == 'success'
-      - name: Analyze code
-        run: dart analyze --fatal-infos
-        if: always() && steps.install.outcome == 'success'
+      - run: dart format --output=none --set-exit-if-changed .
+        if: matrix.sdk == 'dev' && steps.install.outcome == 'success'
+      - run: dart analyze --fatal-infos
+        if: matrix.sdk == 'dev' && steps.install.outcome == 'success'
+      - run: dart analyze
+        if: matrix.sdk == '2.12.0' && steps.install.output == 'success'
```

```suggestion
        if: matrix.sdk != 'dev' && steps.install.output == 'success'
```

Ah I'm sorry, I just remember the other way I had decided to handle this. `!= 'dev'` would allow this to scale if we ever introduce another SDK to the matrix, like if we did do `[2.12.0, stable, dev]` for some reason.